samples/samples-js/AddProductParams/index.js

```diff
@@ -0,0 +1,16 @@
+module.exports = async function (context, req) {
+    context.log('JavaScript HTTP trigger function processed a request.');
+
+    const newProduct = {
+        "productid": req.params?.productid ?? null,
```

Comment:
`?? null` isn't really doing anything - it's already null (that's what the ?? operator is checking)